test/v2/load/load_generator.go

```diff
@@ -143,9 +145,28 @@ func (l *LoadGenerator) Stop() {
 
 // run runs the load generator.
 func (l *LoadGenerator) run() {
-	ticker := time.NewTicker(l.submissionPeriod)
+
+	// Start with frequency 0.
+	ticker, err := common.NewVariableTickerWithFrequency(l.ctx, 0)
+	if err != nil {
+
+	}
+
+	defer ticker.Close()
+	// Set acceleration prior to setting target frequency, since acceleration 0 allows "infinite" acceleration.
+	err := ticker.SetAcceleration(l.config.FrequencyAcceleration)
```

Comment:
I'm confused how this is passing CI, my IDE has an error `no new variables on left side of :=`